modules/ROOT/pages/errors/gql-errors/52N35.adoc

```diff
@@ -0,0 +1,4 @@
+= 52N35
+
+== Status description
+error: procedure exception - procedure compilation failed. Failed to compile procedure/function defined in `{ $procClass }`: `{ $msg }`
```

Comment:
```suggestion
error: procedure exception - procedure compilation failed. Failed to compile procedure defined in `{ $procClass }`: `{ $msg }`
```

I changed this one when created 53N35, but missed to update in https://github.com/neo4j/docs-status-codes/pull/243